### ISSUE TITLE
[🐛 Bug]: Missing WebDriver Docker image 4.19.1 contrary to established conventions

### ISSUE THREAD

**Issue by eliasbalasis** [SeleniumHQ/docker-selenium#2178]
### What happened?

The WebDriver Docker Image for Selenium 4.19.1 is missing from Docker Hub.

This is contrary to standard conventions on which certain tools depend.

for example: Selenium 4.16.1 has published WebDriver Docker Images with tag 4.16.1 while Selenium 4.19.1 has not.

### How can we reproduce the issue?

```shell
check https://hub.docker.com/r/selenium/standalone-chrome

Notice the presence of 4.16.1 tag and the absence of 4.19.1 tag

The same holds true for the whole WebDriver Docker Images group.
```

### Relevant log output

```shell
Not directly relevant

Caused by: java.lang.RuntimeException: org.testcontainers.containers.ContainerFetchException: Can't get Docker image: RemoteDockerImage(imageName=selenium/standalone-chrome:4.19.1, imagePullPolicy=DefaultPullPolicy(), imageNameSubstitutor=org.testcontainers.utility.ImageNameSubstitutor$LogWrappedImageNameSubstitutor@144ee8a7)
Caused by: org.testcontainers.containers.ContainerFetchException: Can't get Docker image: RemoteDockerImage(imageName=selenium/standalone-chrome:4.19.1, imagePullPolicy=DefaultPullPolicy(), imageNameSubstitutor=org.testcontainers.utility.ImageNameSubstitutor$LogWrappedImageNameSubstitutor@144ee8a7)
Caused by: com.github.dockerjava.api.exception.NotFoundException: 
Status 404: {"message":"manifest for selenium/standalone-chrome:4.19.1 not found: manifest unknown: manifest unknown"}
```

### Operating System

Linux

### Selenium version

4.19.1

### What are the browser(s) and version(s) where you see this issue?

Chrome

### What are the browser driver(s) and version(s) where you see this issue?

4.19.1

**Comment by VietND96** [comment]
Actually, it still needs to trigger a release here manually once upstream has a new server jar package.
This ticket will probably implement a mechanism to automatically bump a new image tag whenever a new jar version is detected on the upstream repo.
@diemol, do you think that's fine?

**Comment by eliasbalasis** [comment]
Thanks @VietND96.

It would be perfect if the automatic mechanism you are referring to could publish Docker images with same tag as the Selenium verion, so that compatibility with 3rd party tools that rely on this convention won't be broken.

**Comment by VietND96** [comment]
It would take some time to be done, so I will keep this ticket open till then.
To unblock others downstream, I recently triggered to publish `4.19.1`

**Comment by diemol** [comment]
We could do something where a PR is created automatically when a new upstream version is released. We have several dependencies that it is preferable to have someone do the merge and trigger the release.

**Comment by eliasbalasis** [comment]
Thanks @diemol.

I assume this is how Selenium 4.16.1 has done the same, or has the convention changed inadvertently ?

**Comment by diemol** [comment]
No, it is just that we released 4.19.1, which took a couple of days to release the docker image. This is a project run by volunteers and sometimes things simply take longer.

**Comment by eliasbalasis** [comment]
Thanks @diemol , I understand.
